packages/connect/src/events/ui-request.ts

```diff
@@ -23,6 +23,12 @@ export const UI_REQUEST = {
     FIRMWARE_NOT_COMPATIBLE: 'ui-device_firmware_not_compatible',
     FIRMWARE_NOT_INSTALLED: 'ui-device_firmware_not_installed',
     FIRMWARE_PROGRESS: 'ui-firmware-progress',
+
+    /** connect is waiting for device to automatically disconnect */
+    FIRMWARE_DISCONNECT: 'ui-firmware_disconnect',
+    /** connect is waiting for device to be automatically reconnected */
+    FIRMWARE_RECONNECT: 'ui-firmware_reconnect',
```

Comment:
maybe these should ge more generic? maybe `DEVICE_REQUEST_DISCONNECT` and `DEVICE_REQUEST_RECONNECT`